website/docs/docs/dbt-cloud/on-premises/setup.md

```diff
@@ -69,7 +69,7 @@ You can apply or roll back these changes at any time by clicking the Deploy and
 
 ### Github Setup
 
-This setup is only required for Github App usage. 
+This setup is only required for Github App usage. Please reference the [Prerequisites](https://docs.getdbt.com/docs/dbt-cloud/on-premises/prerequisites/#github) for necessary information. 
```

Comment:
you should be able to use `on-premises/prerequisites#github` as the URL here. This will make the link work in local dev, and will turn on link validation :)

Reply:
Ah epic, thank you!